INSTALL.md

```diff
@@ -186,6 +186,10 @@ can then load the core in GDB.
 
 ## Install distribution
 
+<a href="https://repology.org/project/rofi-wayland/versions">
```

Comment:
Can you rewrite it to use markdown instead of inline html (like images in README.md)?

Reply:
GitHub Markdown doesn’t allow to float and align images, that’s the reason why I used inline HTML instead of the Markdown image syntax.